libmscore/measure.cpp

```diff
@@ -1440,6 +1443,11 @@ Element* Measure::drop(EditData& data)
                   score()->undoAddElement(e);
                   return e;
 
+            case ElementType::MEASURE_NUMBER:
+                  undoChangeProperty(Pid::MEASURE_NUMBER_MODE, (int) MeasureNumberMode::SHOW);
```

Comment:
Can you please avoid those old style C casts? At least use the C++ function style casts `int(MeasureNumberMode::SHOW)`